### DISCUSSION TITLE
own "team" assigned automatically by creating a new project by user / team leader

### DISCUSSION THREAD

**Discussion by Flo-MF** [kimai/kimai#3405] [General]
We require the functionality, that team leaders and even team members can create own projects.
That is working so far - but when they create a project, everybody, who use Kimai, can see this project.

Is it possible, that the created projects will be limited to its own teams, from which the user is a member by default?

I hope, it's clear, what we require.
Thanks in advance.

**Comment by kevinpapst** [comment]
Yes, understand what you want. But that functionality doesn't fit into Kimai core. 
Nevertheless can be developed as custom plugin, drop me a mail if you are interested.
